.github/workflows/arc-publish-chart.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
         python-version: '3.11'
 
     - name: Set up chart-testing
-      uses: helm/chart-testing-action@v2.3.1
+      uses: helm/chart-testing-action@v2.6
```

Comment:
Unrelated change; the bump is only required because a the dependency of this action (sigstore/cosign) has moved and cannot be downloaded with 2.3.1 anymore.